SwissTransferCoreUI/Components/AddFilesMenuView.swift

```diff
@@ -0,0 +1,124 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCore
+import PhotosUI
+import STResources
+import SwiftUI
+import SwissTransferCore
+
+public struct AddFilesMenuView<Content: View>: View {
+    @State private var showImportFile = false
+    @State private var showCamera = false
+
+    // Library
+    @State private var showLibrary = false
+    @State private var selectedPhotos: [PhotosPickerItem] = []
+
+    private let completion: ([URL]) -> Void
+    private let label: () -> Content
+
+    public init(completion: @escaping ([URL]) -> Void, @ViewBuilder label: @escaping () -> Content) {
+        self.completion = completion
+        self.label = label
```

Comment:
We should call the viewBuilder function here.
It helps SwiftUI to better understand the dependencies of the Content view and update it when necessary.